packages/liferay-js-publish/src/index.js

```diff
@@ -50,27 +50,23 @@ async function main() {
 
 	git('push', remote, 'master');
 
-	await confirm('Merge "master" into "stable"?');
+	if (isPrereleaseVersion(pkg)) {
+		print('🛈 Detected prerelease version: skipping merge/push to stable');
+	} else {
+		await confirm('Merge "master" into "stable"?');
 
-	git('checkout', 'stable');
+		git('checkout', 'stable');
 
-	git('merge', '--ff-only', 'master');
+		git('merge', '--ff-only', 'master');
 
-	await confirm(`Push to ${remote}/stable?`);
+		await confirm(`Push to ${remote}/stable?`);
 
-	git('push', remote, 'stable', '--follow-tags');
+		git('push', remote, 'stable', '--follow-tags');
+	}
```

Comment:
:+1:

Reply:
Can we do that in the previous push: `git push upstream master`? I guess that should fix it...

Reply:
You could.

The reason why I did it with the `stable` push originally was just to defer it until as late as possible (in case something else failed).

I think doing it with the `master` push instead is reasonable.

Reply:
I'll do that then.

Reply:
In fact, if the merge fails it's because we have pushed something to stable and not master, which shouldn't happen. And even if it happens, it would probably need manual fixing. 

In that case, the tag could be removed manually while fixing the mess and/or preventing the multiverse from collapsing too.